qemu/tests/qemu_guest_agent_snapshot.py

```diff
@@ -41,18 +48,37 @@ def _action_before_fsfreeze(self, *args):
             dict(verbose=True, timeout=copy_timeout))
         self.bg.start()
 
+    def check_snapshot(self):
+        """
+        Check whether the snapshot is created successfully.
+        """
+        snapshot_info = str(self.vm.monitor.info("block"))
+        snapshot_node_name = self.params.get("snapshot_node_name")
+        if self.params.get("snapshot_file") not in snapshot_info:
+            self.test.fail("Snapshot doesn't exist:%s" % snapshot_info)
```

Comment:
I think should add one step to display checking result like "Found snapshot in guest ".
I can display log,so tester can know what happened in checking.

Reply:
finish editing